.gitmodules

```diff
@@ -293,3 +293,11 @@
 	path = third_party/imgui/repo
 	url = https://github.com/ocornut/imgui
 	platforms = linux
+[submodule "third_party/asr/asr582x/asr_sdk"]
+	path = third_party/asr/asr582x/asr_sdk
+	url = https://github.com/asriot/ASR582X_Freertos.git
+	branch = matter
```

Comment:
can we add some "platforms" support here so these are not checked out for every platform?